testnet/stacks-node/src/config.rs

```diff
@@ -776,6 +776,14 @@ impl Config {
         path
     }
 
+    /// Returns the path `{get_chainstate_path()}/estimates`, and ensures it exists.
+    pub fn get_estimates_path(&self) -> PathBuf {
+        let mut path = self.get_chainstate_path();
+        path.push("estimates");
+        assert!(fs::create_dir_all(&path).is_ok());
```

Comment:
@kantai note the call to `create_dir_all`

What do you think about this location for it?

Reply:
Ah, oops, sorry I missed this -- yeah, this works.

Reply:
Though, I would suggest changing from the assert to an expect:

```suggestion
        fs::create_dir_all(&path).expect("Failed to create cost estimates path in node chainstate directory.");
```

Reply:
updated